src/main/scala/edu/ie3/simona/model/participant/evcs/EvcsChargingProperties.scala

```diff
@@ -39,7 +39,7 @@ trait EvcsChargingProperties {
   ): Power = {
     val evPower = currentType match {
       case ElectricCurrentType.AC =>
-        ev.pRatedAc
+        ev.sRatedAc
```

Comment:
As we expect evPower in Simona to be active power, this schould be sRatedAc * cosPhi in my point of view